miden-lib/asm/utils/asset_utils.masm

```diff
@@ -0,0 +1,193 @@
+use.utils::account_utils
```

Comment:
This import may cause an issue with my suggestions form the last comments. But I wonder if we can get rid of it. There are 3 procedures in this module which rely on `account_utils` and as far as I can tell, these procedures are not "shared". So, if we move them back into the kernel `asset` module, we could get rid of this import.